src/custom/space.ts

```diff
@@ -9,6 +9,7 @@ export interface DatabaseSpace {
   name: string;
   domain: string;
   token: string;
+  privateToken?: string;
```

Comment:
This also needs to be added to `NacelleSpace` & `SpaceInput` in `type-defs.ts` and then `npm run generate` should be re-run to update the generated TypeScript types